googleapis/pubsub/v1/pubsub.pb.go

```diff
@@ -1373,8 +2084,9 @@ var _ grpc.ClientConn
 // is compatible with the grpc package it is being compiled against.
 const _ = grpc.SupportPackageIsVersion4
 
-// Client API for Subscriber service
-
+// SubscriberClient is the client API for Subscriber service.
+//
+// For semantics around ctx use and closing/ending streaming RPCs, please refer to https://godoc.org/google.golang.org/grpc#ClientConn.NewStream.
```

Comment:
This is the main change I was going for. (see my change to the grpc proto generator https://github.com/golang/protobuf/commit/32a84b27e28ab9f681f0df16160c4ef1f6587094)

Reply:
@jba small note - this (belated) CL will help in future cases where we/users are confused about closing streams.

(link - https://godoc.org/google.golang.org/grpc#ClientConn.NewStream)